### ISSUE TITLE
A guide for refreshing the search interface

### ISSUE THREAD

**Issue by Haroenv** [algolia/vue-instantsearch#366]
https://github.com/algolia/vue-instantsearch/issues/241#issuecomment-320491382

a custom component with searchStore.refresh()

**Comment by abattenburg** [comment]
@Haroenv has this feature been added? I know it's not in the docs yet, but I was just trying to use `searchStore.refesh()` in a method for testing purposes and it doesn't seem to have any effect. 

I would love to see even a little example. Being able to refresh algolia without a hard reset to the entire component would be amazing.

**Comment by abattenburg** [comment]
Never mind made a silly mistake and it's working just fine. Great to know this feature has been added!

**Comment by Haroenv** [comment]
Feel free to leave a short snippet here for the time that it's not documented explicitly yet

**Comment by abattenburg** [comment]
Never mind it looks like I celebrated too soon. I'm not sure if I never had it working in the first place, but when I went to post the example it no longer works. The `store.refresh() ` was added to the vue package as well as the react one right?

I'll post my setup and questions to a different issue.

**Comment by Haroenv** [comment]
@abattenburg, react IS has refresh with a different method, but there’s a guide about it. Thanks for posting a new issue

**Comment by gqoew** [comment]
Hi @Haroenv 

Thank you for updating this guide. I was wondering if there is an alternative way create a `method()` to refresh the search interface? It would allow to refresh results without having to create a new component and to refresh only after a set of user actions success (add, delete, update documents in firestore), instead of having a refresh button or a periodic update.

Something like:

```javascript
methods: {
  refreshResults () {
      store.clearCache()
      store.refresh()
  },
```

I use a basic config:

```javascript
      <ais-instant-search :search-client="searchClient" index-name="index">
        <ais-search-box/>
        <ais-hits>
        ...
        </ais-hits>
      </ais-instant-search>
    </div>
```

Not sure how to access the `store` though.

Thanks!

**Comment by jboomer007** [comment]
was this ever considered i am needing to od the same thing have a method to reshest the cache not tied to a button.

**Comment by Haroenv** [comment]
You can access it programmatically as well, like in this guide https://www.algolia.com/doc/guides/building-search-ui/going-further/improve-performance/vue/#refresh-the-cache-periodically @jboomer007, or do you mean something else?

**Comment by Haroenv** [comment]
I don't understand what you mean, sorry @jboomer007, if you have an example somewhere that would help
